test/_init_tests.js

```diff
@@ -160,6 +166,10 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    return this.if(this.platform.isIe.bind(this.platform));
```

Comment:
Not sure this will work. Probably need to remove the `skipIe()` function from `skipMatchers` array.